src/pybind/mgr/dashboard/frontend/src/app/shared/shared.module.ts

```diff
@@ -2,11 +2,17 @@ import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 
 import { ApiModule } from './api/api.module';
+import { AuthService } from './api/auth.service';
```

Comment:
Please remove the changes in this file. Each service should be provided in the respective `api` or `services` module.